example.csharp/addons/tiltfive/scenes/T5ControllerCS.cs

```diff
@@ -0,0 +1,80 @@
+using Godot;
+using System;
+
+[Tool]
+public partial class T5ControllerCS : T5NodeCS
+{
+	public delegate void ButtonEvent(T5ControllerCS self, StringName name);
+	public delegate void FloatEvent(T5ControllerCS self, StringName name, float value);
+	public delegate void Vector2Event(T5ControllerCS self, StringName name, Vector2 value);
+
+	public event ButtonEvent ButtonPressed;
```

Comment:
This uses the normal C# event dispatching system rather than the Godot [Signal] system. As a result these bypass the normal Godot machinery for unsubscribing when a node is removed from the scene tree.